WebDevStudios/nextjs-wordpress-starter
# Feature/tailwind defaults

Closes N/A

### Link

Link to the branch preview generated by Vercel.

### Description

* Revert tailwind config to defaults
* Go through each component and make sure they still work

### Screenshot

If possible, add some screenshots of your feature.

### Verification

How will a stakeholder test this?

1. Build the site
2. Make sure there are no css / tailwind errors
